docs/content/en/docs/reference/tracing-policy.md

```diff
@@ -1308,6 +1310,29 @@ The above would be executed in the kernel as
 arg != arg0 AND arg != arg1
 ```
 
+The operator type `And` performs and bitwise operation on the argument value
+and defined values. The argument type needs to be one of the value types.
+
+For example in following YAML snippet we match second argument for bits
+1 and 9 (0x200 value). We could use single value 0x201 as well.
+
+```yaml
+matchArgs:
+- index: 2
+  operator: "And"
+  values:
+  - 1
+  - 0x200
+```
+
+The above would be executed in the kernel as
+```yaml
+arg & 1 OR arg & 0x200
+```
```

Comment:
Thanks adding the docs!
It's somewhat confusing that the different "AND" values are OR-ed, but it makes sense and it's great that you clarified in the docs.

Reply:
Thank you very much @olsajiri for making this. It is also a bit confusing. The doc makes it clear.

However right now we have the logical `OR` and `AND` implicitly being used in our selectors and other match fields in case we have a CRD with multiple selectors or a selector with multiple matches.

Example the `OR` and `AND` are implicitly implied as in `Equal or NotEqual` for `matchArgs`.

So how about we make it clear that this is bitwise AND with `&` instead of "And" ?   most people who may use this are technical enough to know the flags so using `&` seems fine

This also saves the `OR` and `AND` so they can be combined maybe later with the `&` , we can then change the default behaviour from `AND` to be `OR` as it is now. Not sure about this last point, but safe keeping `OR` and `And` can allow to do perform such things...

Reply:
ok, but I'm not sure using '&' as non-alpha char might bring other issues,
how about using 'Mask' instead? @tixxdz

Reply:
> ok, but I'm not sure using '&' as non-alpha char might bring other issues, how about using 'Mask' instead? @tixxdz

I see yep, could be problematic in tracingpolicies, I don't have that much clue on it. For 'mask' hmm not sure if it can be interpreted for other bitwise operations...

Reply:
I see 'Mask' as better fit than 'And' ... I think mask is always used like 'value & mask', if I'd think of that before I wouldn't use 'And' probably ;-)

Reply:
`BitwiseAND` would be another option.

Reply:
'BitwiseAND' is fine, but 'mask' is easy writing! what about document that by **default** 'mask' is 'value & mask', and if there are other user bitwise cases: like 'or', 'xor', etc we can add a "not required" property to 'MatchArgs', like: 'bitmask: ' to tune the mask and it is about bits?